File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& a) {
        unordered_map<int,int>map;
        unordered_set<int>set;

        for(auto x:a) map[x]++;

        for(auto x:map){
            if(set.find(x.second)!=set.end()) return 0;
            set.insert(x.second);
        }

        return 1;
    }
};

File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.py
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        d={}
        for i in arr:
            if(i in d):
                d[i]+=1
            else:
                d[i]=1
        l=list(d.values())
        if(len(l)==len(set(l))):
            return True
        return False
        